Zotero/Scenes/Detail/PDF/Models/PDFSettings.swift

```diff
@@ -35,12 +44,14 @@ extension PDFSettings: Codable {
         let appearanceRaw = try container.decode(UInt.self, forKey: .appearanceMode)
         let modeRaw = (try? container.decode(UInt.self, forKey: .pageMode)) ?? 2
         let fittingRaw = (try? container.decode(Int.self, forKey: .pageFitting)) ?? 2
+        let isFirstPageAlwaysSingle = (try? container.decode(Bool.self, forKey: .isFirstPageAlwaysSingle)) ?? false
```

Comment:
We should use the PSPDFKIt default here, which is `true`.

Reply:
You're right, I got that mixed up in here, got it right just few lines above 😅